Libraries/ActionSheetIOS/NativeActionSheetManager.js

```diff
@@ -12,7 +12,7 @@
 
 import type {TurboModule} from '../TurboModule/RCTExport';
 import * as TurboModuleRegistry from '../TurboModule/TurboModuleRegistry';
-import type {NativeOrDynamicColorType} from '../StyleSheet/NativeOrDynamicColorType'; // TODO(macOS ISS#2323203)
```

Comment:
Are these changes still forked from core?

Reply:
These definitely still seem to be https://github.com/facebook/react-native/blob/0.63-stable/Libraries/ActionSheetIOS/NativeActionSheetManager.js#L26.

There were others where things seemed a little more confusing to me that I wanted to check with @tom-un, e.g. Libraries/Components/Switch/AndroidSwitchNativeComponent.js

Reply:
Ah. Yeah. I think that's because our native module still doesn't support the native color and only supports the number format. We have an invariant in the JS that forces it to be a number and throws on native colors: https://github.com/facebook/react-native/blob/0.63-stable/Libraries/ActionSheetIOS/ActionSheetIOS.js#L72-L75